Phone_2UPD.py

```diff
@@ -0,0 +1,50 @@
+class Employee:
+
+    def __init__(self, name, daily_salary):
+        self.name = name
+        self.daily_salary = daily_salary
+
+    def work(self):
+        return "I come to the office."
+
+    def __str__(self):
+        return f"Employee: {self.name}"
+
+    def __lt__(self, other):
+        return self.daily_salary < other.daily_salary
+
+    def __gt__(self, other):
+        return self.daily_salary > other.daily_salary
+
+    def __eq__(self, other):
+        return self.daily_salary == other.daily_salary
+
+
+
+class Recruiter(Employee):
+    def work(self):
+        return "I come to the office and start to hiring."
```

Comment:
the phrase `I come to the office` is already specified in line 8, try to extend existing method